fdbbackup/FileDecoder.actor.cpp

```diff
@@ -452,10 +457,34 @@ class DecodeProgress {
 			}
 
 			// Decode a file block into log_key and log_value chunks
-			Standalone<VectorRef<KeyValueRef>> chunks =
+			state Standalone<VectorRef<KeyValueRef>> chunks =
 			    wait(fileBackup::decodeMutationLogFileBlock(self->fd, self->offset, len));
 			self->blocks.push_back(chunks);
 
+			if (self->save) {
```

Comment:
I am not sure about this code, but seems irrelevant?

Reply:
This is not related to knobs, and is some improvement for `fdbdecode`.